Rpackage/tests/testthat/test-cv.R

```diff
@@ -0,0 +1,26 @@
+library(mmit)
+library(testthat)
+
+data(neuroblastomaProcessed, package="penaltyLearning")
+
+set.seed(1)
+n.folds <- 5L
+fold.vec <- sample(rep(1:n.folds, l=nrow(neuroblastomaProcessed$target.mat)))
+test.fold <- 1L
+is.train <- fold.vec != test.fold
+X.train <- data.frame(neuroblastomaProcessed$feature.mat[is.train,])
+y.train <- neuroblastomaProcessed$target.mat[is.train,]
+X.test <- data.frame(neuroblastomaProcessed$feature.mat[!is.train,])
+y.test <- neuroblastomaProcessed$target.mat[!is.train,]
+
+test_that("mmit.cv(X,y) works", {
+  fit <- mmit.cv(X.train, y.train)
+  pred.vec <- mmit.cv.predict(fit, X.test)
```

Comment:
@parismita please change this test back to what I proposed (using the predict S3 method is what an R user should be able to do), e.g.
```
fit <- mmit.cv(X.train, y.train)
pred.vec <- predict(fit, X.test)
```

Reply:
so if we define predict.mmit.cv, we can call it by directly using predict?

Reply:
right.

whenever you do predict(x) that is equivalent to predict.class_of_x(x)

Reply:
if you look at the definition of the predict function, it just calls UseMethod("predict") which handles the "method dispatch" -- it figures out what class your object is, and calls the corresponding predict.* function

Reply:
@tdhock  should I do this for mmif as well and change mmif.predict to predict?

only mmit must return a party object of trees...thus we cant do "predict" directly on it.

Reply:
shall i change each predict function to predict.* for uniformity, then we can define mmif return object as s3 class as well.

Reply:
right